### PR TITLE
Make teller-track-spending executable and give human readable name

### LINKED ISSUE #129: re: teller-track-spending
FAO: (@sebinsua)

Can you changes the permissions on the file to make it executable and submit an update? Also, please rename in metadata to nice human readable name. :)

### PR DESCRIPTION
**Pull request by sebinsua**
Should fix #129.

### CODE REVIEW

File: Finance/teller-track-spending.1h.sh [hunk 1]
```diff
@@ -7,7 +7,7 @@
 # - teller-cli: https://github.com/sebinsua/teller-cli#from-release
 # - pcregrep: `brew install pcre`
 #
-# <bitbar.title>teller-track-spending</bitbar.title>
+# <bitbar.title>Track spending at UK banks</bitbar.title>
```
Comment (keithamus) [inline Finance/teller-track-spending.1h.sh]: Perhaps this should still mention Teller?

Comment (sebinsua) [inline Finance/teller-track-spending.1h.sh]: I can do that!

I wasn't sure how long it could be to be honest - so I was erring on side of caution.

Comment (sebinsua) [inline Finance/teller-track-spending.1h.sh]: Can I put a hyperlink in the description actually? Might be a good idea to draw attention to signup.

Comment (keithamus) [inline Finance/teller-track-spending.1h.sh]: We already have a precedent of some fairly long descriptions, see [the pull-requests script](https://github.com/matryer/bitbar-plugins/blob/master/Dev/GitHub/pull-requests.5m.sh#L7) :smile: .

Comment (sebinsua) [inline Finance/teller-track-spending.1h.sh]: Wow a blob of JavaScript inside a shell script, too. hahaha.

Comment (keithamus) [inline Finance/teller-track-spending.1h.sh]: Actually I just realised you don't have a description! Perhaps the title could be `Teller Spending Tracker` and the desc could be `Track your spending with the teller-cli and the Teller banking api`, or something similar?

**Comment by sebinsua** [comment]
Updated RE: your comments.

**Comment by keithamus** [comment]
:hand: :+1: :ok_hand:

**Pull request merged**